client/build/api.js

```diff
@@ -0,0 +1,78 @@
+"use strict";
+
+exports.getTasks = getTasks;
+exports.getTaskById = getTaskById;
+exports.addTask = addTask;
+exports.editTask = editTask;
+exports.deleteTask = deleteTask;
+Object.defineProperty(exports, "__esModule", {
```

Comment:
That "Object.defineProperty"? Files in 'build' directory are created automatically by systemjs-builder from ES6, so I'm not sure either. Maybe it's needed for handling modules import.